sdk/translation/azure-ai-translation-document/tests/test_model_updates.py

```diff
@@ -0,0 +1,253 @@
+            "last_updated_on": datetime.datetime.now(),
+            "status": "Succeeded", 
+            "summary": {"total": 10, "success": 10, "failed": 0, "in_progress": 0, "not_yet_started": 0, "canceled": 0, "total_characters_charged": 5000},
+            "error": None 
+        }
+        translation_status_dict = TranslationStatus(**params)
+        self.validate_translation_status(translation_status_dict)
+
+    def validate_translation_target(self, translation_target):
+        assert translation_target is not None
+        assert translation_target.target_url is not None
+        assert translation_target.language is not None
+        assert translation_target.category_id is not None
+        assert translation_target.glossaries and translation_target.glossaries[0].glossary_url and translation_target.glossaries[0].file_format is not None
+        assert translation_target.storage_source is not None
+
+    def validate_document_translation(self, document_translation):        
+        assert document_translation is not None
+        assert document_translation.source_url is not None
+        assert document_translation.targets and document_translation.targets[0].target_url and document_translation.targets[0].language is not None
+        assert document_translation.source_language is not None
+        assert document_translation.storage_type is not None
+        assert document_translation.storage_source is not None
+        assert document_translation.prefix is not None
+        assert document_translation.suffix is not None
+
+    def validate_translation_glossary(self, translation_glossary):        
+        assert translation_glossary is not None
+        assert translation_glossary.glossary_url is not None
+        assert translation_glossary.file_format is not None
+        assert translation_glossary.format_version is not None
+        assert translation_glossary.storage_source is not None
+
+    def validate_document_status(self, document_status):
+        assert document_status is not None
+        assert document_status.source_document_url is not None
+        assert document_status.created_on is not None
+        assert document_status.last_updated_on is not None
+        assert document_status.status is not None
+        assert document_status.translated_to is not None
+        assert document_status.translation_progress is not None
+        assert document_status.id is not None
+        assert document_status.translated_document_url is not None
+        assert document_status.characters_charged is not None
+
+    def validate_translation_status(self, translation_status):
+        assert translation_status is not None
+        assert translation_status.id is not None
+        assert translation_status.created_on is not None
+        assert translation_status.last_updated_on is not None
+        assert translation_status.status is not None
+        assert translation_status.summary is not None
```

Comment:
we should verify that we can still access the old named attributes for this class: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/translation/azure-ai-translation-document/azure/ai/translation/document/models/_patch.py#L387-L393

Reply:
Updated